Source/WebCore/inspector/agents/InspectorCSSAgent.cpp

```diff
@@ -927,6 +927,16 @@ Inspector::Protocol::ErrorStringOr<Ref<JSON::ArrayOf<Inspector::Protocol::CSS::C
             }
             if (values->length())
                 property->setValues(WTFMove(values));
+        } else {
+            // For properties that aren't keyword-fast-path eligible (e.g., display),
```

Comment:
can we add maybe a test case for something like `display` in `LayoutTests/inspector/css/getSupportedCSSProperties.html`?